src/app/routes.tsx

```diff
@@ -117,10 +117,10 @@ const routes: IAppRoute[] = [
   {
     component: CreateCache,
     exact: true,
-    label: 'Caches',
+    label: 'Cache Wizard',
```

Comment:
@dvagnero I don't like the title but I'm unable to think in a better name. Any ideas ?

Reply:
Hey @karesti , I think the term wizard is widely known but here it's wizard + the possibility to provide a cache configuration. So I'm thinking about "Cache Configuration" or  "Cache Setup". Later today I'm giving demo to Erin from the UX team so I'll check with her

Reply:
Hey @dvagnero any update on the name?

Reply:
Apologies for the delay. I did mention this change but didn't follow up on it. @edonehoo do you have any suggestions?

Reply:
I was actually going to suggest "Cache Setup" before seeing that you offered that idea! I think it's best to give context to what the wizard is used for, which "setup" seems to cover. "Create Cache" may also work, if clicking on that tab will always initiate the creation of a new cache (rather than continuing a cache setup that was previously started).

Reply:
I also like "Cache setup", if everyone agrees moving ahead with that :+1: